book/src/keymap.md

```diff
@@ -118,6 +118,10 @@
 | `Alt-K`  | Remove selections matching the regex                              | `remove_selections`                  |
 | `$`      | Pipe each selection into shell command, keep selections where command returned 0 | `shell_keep_pipe`     |
 | `Ctrl-c` | Comment/uncomment the selections                                  | `toggle_comments`                    |
+| `Alt-k`  | Expand selection to parent syntax node                            | `expand_selection`                   |
+| `Alt-j`  | Shrink syntax tree object selection                               | `shrink_selection`                   |
+| `Alt-h`  | Select previous sibling node in syntax tree                       | `select_prev_sibling`                |
+| `Alt-l`  | Select next sibling node in syntax tree                           | `select_next_sibling`                |
```

Comment:
This seemed a bit weird to me, shouldn't it be hl to expand/shrink and hj to navigate between sibling?

Reply:
This makes sense to me. If you picture the tree vertically, with expand, you are going up the tree, shrink you are going down. Next sibling is moving laterally to the right, and previous is left.

Reply:
I think about it the same way :point_up: 

I could see it going the other way too because often `A-l` will move your selection downwards (to the next function block for example), but that depends on the size of the selection (e.g. `A-l` will move right in a function's argument list if it's all on one line) which can be a little confusing